src/event/event.c

```diff
@@ -768,7 +768,7 @@ _dispatch_timer_heap_update(dispatch_timer_heap_t dth,
 #define _dispatch_timer_du_debug(what, du) \
 		_dispatch_debug("kevent-source[%p]: %s kevent[%p] { ident = 0x%llx }", \
 				_dispatch_wref2ptr((du)->du_owner_wref), what, \
-				(du), (unsigned long long)((du)->du_ident))
+				(du), (unsigned long long)(du)->du_ident)
```

Comment:
This seems like an unnecessary change ... and I kinda do prefer the original variant.  Does it cause some issue to leave it as is?